solutions/136/01.py

```
@@ -0,0 +1,7 @@
+def singleNumber(nums: List[int]) -> int:
```

Ruff (0.12.2)

Undefined name `List`

(F821)